test/runtests.jl

```diff
@@ -0,0 +1,20 @@
+using PlutoUI
+using Test
+
+@testset "builtins" begin
+    for s in [
+        Slider(1:0.2:3; default=0.4, style="{width=100px;}"),
+        Button(; style="{width=100px;}"),
+        FilePicker(; style="{width=100px;}"),
+        Radio(["A", "B", "C"]; style="{width=100px;}"),
+        Select(["A", "B", "C"]; style="{width=100px;}"),
+        MultiSelect(["A", "B", "C"]; style="{width=100px;}"),
+        CheckBox(; default=true, style="{width=100px;}"),
+        TextField(; style="{width=100px;}"),
+        NumberField(1:0.2:3; default=2, style="{width=100px;}"),
```

Comment:
Why the braces around the `style` values? Quoting [the standard](https://drafts.csswg.org/css-style-attr/#style-attribute) (linked from [here](https://html.spec.whatwg.org/multipage/dom.html#the-style-attribute)):

> The value of the style attribute must match the syntax of the contents of a CSS declaration block (excluding the delimiting braces) [...]

Also wouldn't it be more interesting to test for the exact HTML results instead of just whether or not they're empty?

Reply:
Yes we should not use braces.